admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java

```diff
@@ -246,28 +263,529 @@ public void searchwildcardSearchMasterdata() {
+	}
+	@Test
+	public void searchMasterdataWithoutLangCode13() {
+		sort.setSortType("desc");
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	@Test
+	public void searchMasterdataWithoutLangCode14() {
+		sort.setSortType("asc");
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode(Arrays.asList(filter), Arrays.asList(sort), page, "eng");
+		searchHelper.searchMasterdataWithoutLangCode(RegistrationCenter.class, searchDtoWithoutLangCode, optionalFilterArray);
+	}
+	@Test
+	public void getMissingDataTest(){
+		String tableName = null;
+		String langCode = null;
+		searchHelper.getMissingData(tableName,langCode);
+	}
+	@Test(expected = Exception.class)
+	public void WithoutFilterMasterdata() {
+		searchHelper.searchMasterdata(RegistrationCenter.class, null, optionalFilterArray);
+	}
+	@Test(expected = NullPointerException.class)
+	public void sortQuery01(){
+		List<SearchSort> searchSorts=new ArrayList<>();
+		SearchSort ss=new SearchSort("abc","bgd");
+		searchSorts.add(ss);
+		ReflectionTestUtils.invokeMethod(searchHelper,"sortQuery",null,null,null,searchSorts);
+	}
+	@Test(expected = RequestException.class)
+	public void sortQuery02(){
+		CriteriaBuilder builder = mock(CriteriaBuilder.class);
+		Root root = mock(Root.class);
+		CriteriaQuery criteriaQuery = mock(CriteriaQuery.class);
+		List<SearchSort> searchSorts=new ArrayList<>();
+		SearchSort ss=new SearchSort(null,"ASC");
+		searchSorts.add(ss);
+		ReflectionTestUtils.invokeMethod(searchHelper,"sortQuery",builder,root,criteriaQuery,searchSorts);
+	}
+	@Test
+	public void sortQuery03(){
+		CriteriaBuilder builder = mock(CriteriaBuilder.class);
+		Root root = mock(Root.class);
+		CriteriaQuery criteriaQuery = mock(CriteriaQuery.class);
+		List<SearchSort> searchSorts=new ArrayList<>();
+		SearchSort sort1 = new SearchSort();
+		sort1.setSortField("abc");
+		sort1.setSortType("ASC");
+		searchSorts.add(sort1);
+		ReflectionTestUtils.invokeMethod(searchHelper,"sortQuery",builder,root,criteriaQuery,searchSorts);
+	}
+	@Test
+	public void	setLangCodeTest(){
+		ReflectionTestUtils.invokeMethod(searchHelper,"setLangCode",null,null,null);
+	}
+	@Test (expected = NullPointerException.class)
+	public void setBetweenValue01(){
+		SearchFilter searchFilter = null;
+		ReflectionTestUtils.invokeMethod(searchHelper,"setBetweenValue",null,null,searchFilter);
+	}
+	@Test (expected = IllegalStateException.class)
+	public void setBetweenValue02(){
+		SearchFilter searchFilter = new SearchFilter();
+		searchFilter.setValue("123");
+		searchFilter.setType("filter");
+		ReflectionTestUtils.invokeMethod(searchHelper,"setBetweenValue",null,RegistrationCenter.class,searchFilter);
+	}
+	@Test (expected = IllegalStateException.class)
+	public void setBetweenValue03(){
```

Comment:
Remove unwanted pieces of code and commented code

Reply:
Removed